packages/sdk-ui-ts/src/client/transformers/UiBridgeTransformer.ts

```diff
@@ -346,17 +346,31 @@ export const convertMoonbeamToUiBridgeTransaction = async (
   }
 }
 
-export const convertWormholeToUiBridgeTransaction = async (
-  transaction: WormholeTxResponse,
-): Promise<UiBridgeTransaction> => {
+export const convertWormholeToUiBridgeTransaction = async ({
+  transaction,
+  network,
+}: {
+  transaction: WormholeTxResponse
+  network: Network
+}): Promise<UiBridgeTransaction> => {
+  const isEthereumWhTransfer =
```

Comment:
Why don't we check also by the tx.source/tx.destination?

Reply:
ahh good call! because then we can check explicitly that it's an ethereum WH transfer, as opposed to a peggy eth transfer for example added in latest commit